examples/json_parser/main.rs

```diff
@@ -1,4 +1,4 @@
-use std::{error::Error, collections::HashMap};
+use std::{error::Error, collections::HashMap, backtrace::Backtrace};
```

Comment:
The Backtrace import is no longer needed (make note of compiler warnings, they usually tell you things like this)

Reply:
oops forgot to get rid of that